rust/src/runner/context.rs

```diff
@@ -136,8 +150,11 @@ impl TestContext {
                 bind(fd, &sockaddr)
             }
             //TODO: error type?
-            FileType::Symlink(target) => symlink(target.as_deref().unwrap_or_else(|| Path::new("test")), &path)
```

Comment:
This looks like a formatting change.  It's good practice to put formatting changes and content changes into separate commits.

Reply:
Hmm, yes, I will be more careful next time.